src/client/wieldmesh.cpp

```diff
@@ -580,7 +576,11 @@ void getItemMesh(Client *client, const ItemStack &item, ItemMesh *result)
 
 		rotateMeshXZby(mesh, -45);
 		rotateMeshYZby(mesh, -30);
+	} else {
+		result->mesh = mesh;
+		return;
```

Comment:
Redundant code.

Reply:
True, I don't remember why I added this.